samples/auth0/program.py

```diff
@@ -0,0 +1,34 @@
+"""Demonstrates AutoKitteh’s Auth0 integration for managing tasks via the Auth0 API."""
+
+from datetime import datetime, timedelta, UTC
+import os
+
+from autokitteh.auth0 import auth0_client
+
+
+auth0 = auth0_client("auth_conn")
+
+ROLE_ID = os.getenv("ROLE_ID", "")
+
+
+def assign_role(event):
+    """Entry-point function for a webhook-based workflow assigning a role to a user."""
+    user = event.data.body.form.get("user_id", "")
+    auth0.roles.add_users(ROLE_ID, [user])
+    print(f"Assigned role {ROLE_ID!r} to user {user!r}")
+
+
+def weekly_user_growth(_):
+    """Fetch and display the number of users created in the past week."""
+    # Remove the unit suffix ("d") and parse as an integer.
+    interval_days = int((os.getenv("TIME_INTERVAL", "7d"))[:-1])
+    one_week_ago = (datetime.now(UTC) - timedelta(days=interval_days)).isoformat()
```

Comment:
This is slightly inaccurate, see https://github.com/autokitteh/kittehub/blob/main/reliability/aws_health_monitor/program.py#L67-L69